examples/mplot3d/2dcollections3d.py

```diff
@@ -25,11 +25,11 @@
 # Fixing random state for reproducibility
 np.random.seed(19680801)
 
-x = np.random.sample(20*len(colors))
-y = np.random.sample(20*len(colors))
+x = np.random.sample(20 * len(colors))
+y = np.random.sample(20 * len(colors))
 c_list = []
 for c in colors:
-    c_list.append([c]*20)
+    c_list.extend([c] * 20)
```

Comment:
oh actually i remember now, it is.
this is because 3d scatter no longer implicitly flattens its input (see the last hunk of the diff).